serve.sh

```diff
@@ -1,9 +0,0 @@
-PORT=$1
```

Comment:
In general the README needs to be updated with information about how to see the demo page and how to run the tests